Symbolica.LLVMSharp.nuspec

```diff
@@ -7,14 +7,15 @@
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <license type="file">LICENSE.txt</license>
     <icon>Icon.png</icon>
-    <projectUrl>https://github.com/SymbolicaDev/Symbolica.LLVMSharp</projectUrl>
+    <projectUrl>https://github.com/Symbolica/Symbolica.LLVMSharp</projectUrl>
     <description>LLVMSharp packaged from an arbitrary version with interop and libLLVM included.</description>
-    <copyright>Copyright 2021 Symbolica Dev Limited</copyright>
+    <copyright>Copyright 2022 Symbolica Dev Limited</copyright>
     <tags>Symbolica dotnet LLVM libLLVM bindings managed interop native</tags>
     <repository type="git" url="https://github.com/microsoft/LLVMSharp" />
     <dependencies>
       <group targetFramework=".NETCoreApp3.1" />
       <group targetFramework="net5.0" />
+      <group targetFramework="net6.0" />
```

Comment:
Just curious, but do we need to depend on net6.0 if we already depend on something like netstandard or have I misunderstood this?

Reply:
I'm not really sure to be honest. I just copied as closely as possible what `LLVMSharp` themselves produce for their own NuGet packages. My best guess is that you want to be more specific when you don't want to transitively depend on things that are not needed in that particular framework. So since `.NETStandard2.0` needs to bring in `Microsoft.Bcl.HashCode` and `System.Memory` but the other ones don't need them maybe that is what they are going for here. I doubt it makes all that much difference.